.github/workflows/import/daFetch.js

```diff
@@ -22,11 +22,13 @@ async function getImsToken() {
   const data = await response.json();
   return data.access_token;
 }
+let token
 
 export const daFetch = async (url, opts = {}) => {
+  if (token) return token
```

Comment:
Is this intended or used for debugging?

Reply:
Nope, it's a small optimization, since we only need to fetch the token once per run. Previously 1 publish = 1 document to import.

Now we potentially import things...: multiple assets (a pdf, a mp4, another mp4) and then the document. So we can cache & reuse the token